src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -17,15 +17,21 @@ const AppealsV2StatusPage = ({ appeal }) => {
   const { type, details } = status;
   const currentStatus = getStatusContents(type, details);
   const nextEvents = getNextEvents(type);
+
+  // Gates the What's Next and Docket chunks
+  const appealIsClosed = CLOSED_STATUSES.includes(type);
+
   return (
     <div>
       <Timeline events={events}/>
       <CurrentStatus
         title={currentStatus.title}
-        description={currentStatus.description}/>
+        description={currentStatus.description}
+        isClosed={appealIsClosed}/>
       <AlertsList alerts={alerts}/>
-      <WhatsNext nextEvents={nextEvents}/>
-      <Docket {...docket}/>
+      {!appealIsClosed && <WhatsNext nextEvents={nextEvents}/>}
+      {!appealIsClosed && <Docket {...docket}/>}
```

Comment:
Might make sense to throw `WhatsNext` and `Docket` into one conditional since they both render under the same condition, but this is pretty explicit so 🤷‍♂️

Reply:
I was going to do that first, but I couldn't find an adequate variable name for them both, so I decided that might mean they shouldn't be coupled together so closely.

And like you said, this is pretty easy to figure out what's going on.